src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -138,39 +137,5 @@
 
 - content_for :ready_function do
   :plain
-    var anchor = document.location.hash;
-    var result = [];
-    if (result = anchor.match(/^#diff_([\d]+)_n\d+$/)) {
-      var file_index = result[1];
-      $('.nav-tabs:not(.disable-link-generation) .nav-link[href="#changes"]').tab('show');
-
-      $('#revision_details_' + file_index).attr('open', 'open');
-
-      document.location.hash = anchor;
-    }
-    else if (anchor == '#comments-list') {
-      $('#conversation-tab').tab('show');
-
-      document.location.hash = anchor;
-    }
-
-    // TODO: Remove this temporary fix once we load the content every time we change tabs.
-    // It's really important that the information displayed in Conversation is accurate and real-time-ish.
-    $('.diff').on('ajax:complete', function() {
-      $('#request-tabs').find('a#conversation-tab').on('shown.bs.tab', function() {
-        $('.tab-pane#conversation').html('<div class="spinner-border" role="status"><span class="visually-hidden">Loading...</span></div>');
-        window.location.reload();
-      });
-    });
-
-    // Same here, but for Changes tab.
-    $('.timeline').on('ajax:complete', function() {
-      $('#request-tabs').find('a#changes-tab').on('shown.bs.tab', function() {
-        $('.tab-pane#changes').html('<div class="spinner-border" role="status"><span class="visually-hidden">Loading...</span></div>');
-        window.location.reload();
-      });
-    });
-
-  setCollapsible();
-
```

Comment:
We don't need all this JS. It was activating tabs on the page load and fetching reloading the page to fetch live data